brave-intl/publishers
# A couple of visual fixes for the dashboard

## Dashboard fixes

#### Before
![image](https://user-images.githubusercontent.com/5459225/55914379-9c856080-5bac-11e9-94ba-fed90491f1c0.png)

#### After

![image](https://user-images.githubusercontent.com/5459225/55914356-89729080-5bac-11e9-8749-01b83fc07376.png)



#### Features

* Fixed problem where only one chart was being shown
* Fixed issue where referral selection did not work in mobile due to no theme in brave-ui
* Fixed nil channel details
* Fixed background on the partners dashbaord

#### How To Test

1. Pull down, make yourself a partner
2. Go to the dashboard and check things out
3. Add a youtube channel and then active referral promo
4. Run `rails database_updates:mock_data:populate_promo_stats` to generate some referral stats you can view on the homepage
